code/aind_smartspim_stitch/zarr_converter/zarr_converter.py

```diff
@@ -25,13 +25,13 @@
 from numcodecs import blosc
 from skimage.io import imread as sk_imread
 
+import utils
```

Comment:
Please, check [this](https://github.com/AllenNeuralDynamics/aind-smartspim-stitch/blob/code_ocean/code/aind_smartspim_stitch/terastitcher.py#L25) to see how we're using the utility functions.